.github/workflows/set-semgrep-docker-tag.yml

```diff
@@ -0,0 +1,64 @@
+# Workflow to manually set the returntocorp/semgrep:canary docker image
+# to a specific version (usually to point to a past version in case of rollback)
+#TODO: should factorize with revert-semgrep-latest-docker-image.yml
+
+name: Set semgrep docker tag
+
+on:
+  workflow_dispatch:
+    inputs:
+      docker_image:
+        description: "Semgrep docker image to update"
+        type: choice
+        required: true
+        options:
+          - returntocorp/semgrep-dev
```

Comment:
are we still using semgrep-dev? I think we removed it.

Reply:
It seems to still exist on Docker Hub, but you're right that we're not really using it anymore. I wanted to be able to test this workflow against an image that isn't the official semgrep docker image (happy to update if there's a better option). I'll add a comment explaining this, and we can plan to remove the `semgrep-dev` option (or whatever we end up with) entirely once we feel confident w/ this workflow

Reply:
I'm remember now that `semgrep-dev` was for github codespaces (or something), so I should probably pick a different image to test with